Makefile

```diff
@@ -102,7 +102,7 @@ debug:
 	@echo PATCH:"$(PATCH)"
 
 # These tags make sure we can statically link and avoid shared dependencies
```

Comment:
Please, fix the comment as noted in the previous PR

Reply:
removed the comment